binding/Binding/SKBitmap.cs

```diff
@@ -723,6 +786,27 @@ public bool InstallPixels (SKImageInfo info, IntPtr pixels, int rowBytes, SKBitm
 			return SkiaApi.sk_bitmap_install_pixels (Handle, &cinfo, (void*)pixels, (IntPtr)rowBytes, proxy, (void*)ctx);
 		}
 
+		public bool ReadPixels (SKImageInfo info, IntPtr dstpixels, int rowBytes, int x, int y)
+		{
+			if (GetPixels() == null) return false;
```

Comment:
Do we need this check? What does ReadPixels do when there are no pixels?

Reply:
it avoids allocating a SKPixmap object if there are no pixels